tests/Bridge/Symfony/Routing/ApiLoaderTest.php

```diff
@@ -96,6 +99,16 @@ public function testNoMethodApiLoader()
         $routeCollection = $this->getApiLoaderWithResourceMetadata($resourceMetadata)->load(null);
     }
 
+    /**
+     * @expectedException \ApiPlatform\Core\Exception\InvalidResourceException
+     */
+    public function testNoShortNameApiLoader()
+    {
+        $resourceMetadata = new ResourceMetadata();
```

Comment:
Lets me be more precise. The variable assignation is useless. It can be on 2 lines for readability, but without the assignation :)